fabric-networking-api-v1/src/testmod/java/net/fabricmc/fabric/test/networking/play/NetworkingPlayPacketTest.java

```diff
@@ -88,6 +94,22 @@ public static void registerCommand(CommandDispatcher<ServerCommandSource> dispat
 					));
 					ServerPlayNetworking.getSender(ctx.getSource().getPlayer()).sendPacket(packet);
 					return Command.SINGLE_SUCCESS;
+				}))
+				.then(literal("transfer").executes(ctx -> {
+					((ServerTransferable) ctx.getSource().getPlayer().networkHandler).transferToServer("localhost", 25565);
```

Comment:
Set a cookie here, incase the set cookie command was not used.

Reply:
Done